docs/Using Fleet/manage-access.md

```diff
@@ -95,7 +95,7 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 
 \* Applies only to Fleet Premium
 
-\** Global observers can view all queries but the UI and fleetctl only list the ones they can run (**observer can run**).
+\** Global observers can view all queries via the API. The UI and fleetctl only list the ones they can run (**observer can run**).
```

Comment:
Alternatively: We might consider breaking the "View all queries & their reports" permissions into 2 lines, e.g. "View all queries  & their reports via the API" and "View all queries & their reports in the UI and fleetctl", so we can get rid of this note altogether and the permissions are **extra** clear.